docs/framework/interop/com-callable-wrapper.md

```diff
@@ -134,9 +134,12 @@ End Class
 ```  
   
 ```csharp
+// The CLR does not expose a class interface for this type.
+// COM clients can call the members of this class using the methods from the IExplicit interface.
```

Comment:
This comment is in the C# snippet, so readers with Visual Basic as the selected language will miss that comment. Moreover, comments are not translated, so this content won't be localized. @sharpist please consider moving this paragraph into the body of the article. 

The same is applicable to another snippet update.

Reply:
@pkulikov I think these both accompanying comments can be excluded from the code examples. At your discretion. Because the main article already contains a detailed description.

Reply:
@sharpist 
>I think these both accompanying comments can be excluded from the code examples.

If the article contains enough explanations, I also don't see the need in the code comments.

Reply:
@Thraka yes
@sharpist thanks for additional updates